staging/provisioning/windows/kubeletstart.ps1

```diff
@@ -247,12 +246,15 @@ if ($global:NetworkPlugin -eq "kubenet") {
             # Add route to all other POD networks
             Update-CNIConfigKubenetDocker $podCIDR $masterSubnetGW
         }
-
-        # startup the service
-        Invoke-Expression $KubeletCommandLine
     }
     catch {
         Write-Error $_
     }
-
 }
+
+# Start the kubelet
+# Use run-process.cs to set process priority class as 'High'
+Add-Type -Path .\run-process.cs
+$exe = "$global:KubeDir\kubelet.exe"
+$args = ($KubeletArgList -join " ")
+[RunProcess.exec]::RunProcess($exe, $args)
```

Comment:
Can this be made optional so in 1.20+ we can use the new `--windows-priorityclass`?

Reply:
Do you mind if we do that in a follow up PR

Reply:
Sounds good. We can do that when we turn on the flag and enable it in 1.20